### PR TITLE
adds a single pack of experimental medication as a cargo goodie

### PR DESCRIPTION
**Pull request by deathrobotpunch**
## About The Pull Request
title above.
## Why It's Good For The Game
allows people with hereditary manifold sickness to purchase a single pack of big pharma endorsed overpriced medicine.

edit: the 2 pack cargo crate has been increased to 600 credits, to better reflect the goodie case price and because companies taking advantage of people with medical conditions is soul
## Changelog
:cl:
add: big pharma now supplies a single pack of experimental medication as an easy to access (albeit expensive) cargo goodie!
balance: price for the cargo crate with 2 pill bottles of experimental medication has been increased to 600 to better reflect the goodie case price.
/:cl:

### CODE REVIEW

File: code/modules/cargo/goodies.dm [hunk 1]
```diff
@@ -321,3 +321,9 @@
 	cost = PAYCHECK_COMMAND * 18
 	access_view = ACCESS_WEAPONS
 	contains = list(/obj/item/gun/ballistic/shotgun/doublebarrel)
+
+/datum/supply_pack/goody/Experimental_Medication
+	name = "Experimental medication Single-Pack"
+	desc = "A single bottle of interdyne brand experimental medication, Overpriced for your inconvienience!"
```
[the remark this answers is not in the document]
Comment (deathrobotpunch) [reply code/modules/cargo/goodies.dm:327]: sure I can change it

[the remark this answers is not in the document]
Comment (deathrobotpunch) [reply code/modules/cargo/goodies.dm]: changed!

File: code/modules/cargo/goodies.dm [hunk 2]
```diff
@@ -321,3 +321,9 @@
 	cost = PAYCHECK_COMMAND * 18
 	access_view = ACCESS_WEAPONS
 	contains = list(/obj/item/gun/ballistic/shotgun/doublebarrel)
+
+/datum/supply_pack/goody/Experimental_Medication
```
Comment (MrMelbert) [inline code/modules/cargo/goodies.dm:325]:
```suggestion
/datum/supply_pack/goody/experimental_medication
```

File: code/modules/cargo/goodies.dm [hunk 3]
```diff
@@ -321,3 +321,9 @@
...
+	name = "Experimental medication Single-Pack"
```
Comment (MrMelbert) [inline code/modules/cargo/goodies.dm:326]:
```suggestion
	name = "Experimental Medication Single-Pack"
```

File: code/modules/cargo/goodies.dm [hunk 4]
```diff
@@ -321,3 +321,9 @@
...
+	desc = "A single bottle of interdyne brand experimental medication, used for treating people suffering from hereditary manifold disease"
```
Comment (CRITAWAKETS) [inline code/modules/cargo/goodies.dm:327]:
```suggestion
	desc = "A single bottle of Interdyne brand experimental medication, used for treating people suffering from hereditary manifold disease."
```

**Comment by deathrobotpunch** [comment]
I changed the spread type in HMS to be incommunicable disease instead of unspread illness

File: code/datums/diseases/chronic_illness.dm [hunk 5]
```diff
@@ -1,7 +1,7 @@
 /datum/disease/chronic_illness
 	name = "Hereditary Manifold Sickness"
 	max_stages = 5
-	spread_text = "Unspread Illness"
+	spread_text = "noncommunicable disease"
```
Comment (MrMelbert) [inline code/datums/diseases/chronic_illness.dm:4]:
```suggestion
	spread_text = "Non-communicable disease"
```

Comment (deathrobotpunch) [reply code/datums/diseases/chronic_illness.dm:4]: I was just about to change this lmao

**Comment by Jackal-boop** [comment]
It would be very funny if it was only 1 or 2 pills instead of a whole pack. Give them the American Healthcare experience

**Comment by Sadboysuss** [comment]
> It would be very funny if it was only 1 or 2 pills instead of a whole pack. Give them the American Healthcare experience

no one would buy it if the medical experimental medication was unchanged with this.

**Pull request merged**